transaction-pool/Cargo.toml

```diff
@@ -12,4 +12,4 @@ log = "0.4.8"
 smallvec = "1.6.0"
 
 [dev-dependencies]
-ethereum-types = { version = "0.11.0", path = "../ethereum-types" }
```

Comment:
To note, I omitted versions for `path` dev-dependencies in the workspace because then Cargo will automatically strip those when publishing, facilitating the publish process by breaking the local dependency cycle on crates (so no need to manually tinker with toml files just in order to be able to publish).